test/test_automated_install.py

```diff
@@ -490,68 +510,25 @@ def test_FTL_download_aarch64_no_errors(Pihole):
     ''')
     download_binary = Pihole.run('''
     source /opt/pihole/basic-install.sh
-    binary="pihole-FTL-aarch64-linux-gnu"
-    FTLinstall
+    create_pihole_user
+    FTLinstall "pihole-FTL-aarch64-linux-gnu"
     ''')
     expected_stdout = tick_box + ' Downloading and Installing FTL'
     assert expected_stdout in download_binary.stdout
     assert 'error' not in download_binary.stdout.lower()
 
 
-def test_FTL_download_unknown_fails_no_errors(Pihole):
```

Comment:
These tests got dropped somehow?

Reply:
https://github.com/pi-hole/pi-hole/pull/3154/commits/b71dd198ee1139796fd7c44192c344b9e4dfb8e0

That commit took out the ftl download tests.